components/BannerNotificationContainer.tsx

```diff
@@ -122,16 +122,15 @@ export const BannerNotificationContainer: React.FC<
           testId={'downloadingVcFailedPopup'}
         />
       )}
-
-      {bannerNotificationController.isDownloadingSuccess && (
+      {/* {bannerNotificationController.isDownloadingSuccess && (
```

Comment:
If we comment out this block, it will impact success banner on download success. @abhip2565

Reply:
yes, we are taking this as known issue @balachandarg-tw